### ISSUE TITLE
Colour Error if Have 37 Values

### ISSUE THREAD

**Issue by emmahodcroft** [nextstrain/auspice#806]
https://github.com/nextstrain/auspice/blob/70a3ad69d75179d424e4ec711e4be648a52313fb/src/util/colorScale.js#L55

I believe this line should be a less-than `<` rather than less-than-equal-to `<=`?

I have just happened to have exactly 37 values (`domain`), which is exactly the length of the `colors` array. However, since `colors` is 37 is length, it has 0-36 indexes. 

```
    colorList = domain.length <= colors.length ?
       colors[domain.length].slice() :
       colors[colors.length - 1].slice();
```

When one goes to `.slice()` in the next line, you can't get `colors[37]`. 

This should be fixed in `v2` as well.

**Comment by jameshadfield** [comment]
This looks like a bug. @emmahodcroft if you fix in `master` then the fix can be merged into `v2` from there. Alternatively if you send a JSON I can resolve this. Thanks!

**Comment by jameshadfield** [comment]
Closed by #1355
